### DISCUSSION TITLE
[Question] useDialog의 content 속성 관련  문의

### DISCUSSION THREAD

**Discussion by yangSangHoon** [wanteddev/montage-web#25] [Q&A]
### 요약

dialog를 띄울때 content에 text가 아닌 reactNode가 들어갈수 있게 수정이 가능할까요?

### 상세

dialog내용에 react component가 들어가야하는 케이스가 있어서,
content속성의 type을 string이 아닌 ReactNode로 바꿔주실수 있을지 문의 드립니다.

(dialog 컨셉과 맞지 않는 요구사항이면 말씀주세요!~ :) )

**Comment by Sh031224** [comment]
콘텐츠 타입을 ReactNode로 변경하는 것은 어렵지 않으나
콘텐츠 내부 padding, 간격을 조정할 수 있게는 못 할 것 같아요.

디자인팀에서 dialog 사용 케이스가 어느정도 정리되면 반영 해보겠습니다

**Reply by 1000peach** [reply]
상훈님이 작성하신 부분은 아닐 것 같은데,
스퀘어에서 dialog와 유사한 디자인에 ReactNode로 들어가야 하는 사용 케이스 공유 드립니다.

<img width="541" alt="스크린샷 2024-05-30 오후 2 27 14" src="https://github.com/wanteddev/wds/assets/36118545/d9f97edf-2ee6-4afa-87d0-a5bdc1cbed6a">

**Reply by yangSangHoon** [reply]
useToast의 content도 동일하게 ReactNode로 들어갈수 있으면 좋을것 같습니다.

**Reply by Sh031224** [reply]
디자인 케이스와 무관하게 번역키 반환 값이 `ReactNode` 인 경우도 있을 것 같아 다음 배포에 반영해두겠습니다
